client/coral-framework/helpers/plugins.js

```diff
@@ -11,7 +11,7 @@ import {loadTranslations} from 'coral-framework/services/i18n';
 export const pluginReducers = merge(
   ...plugins
     .filter((o) => o.module.reducer)
-    .map((o) => ({...o.module.reducer}))
+    .map((o) => ({[o.module.reducer.name] : o.module.reducer}))
```

Comment:
Hmm. I did that at the beginning but I don't think it's right. First, I think we should leave the name of the function as is, just like we handle our reducers in the rest of the app. And second, what happen if I want to use more reducers?